### DISCUSSION TITLE
Are there any plans to integrate with Kubeflow?

### DISCUSSION THREAD

**Discussion by vecorro** [open-metadata/OpenMetadata#6095] [Ideas]
Hi,

It's great that OpenMetadata already integrates with MLFlow and Airflow, however, Kubeflow is also popular and It'd be useful to have integration with it. Are there any plans or workarounds to achieve that?

**Comment by harshach** [comment]
cc @pmbrull

**Reply by pmbrull** [reply]
hi @vecorro, we can look into preparing that integration. Is this something you'd be willing to contribute? We could give you a hand on getting started with the code.

Thanks!
